### PR TITLE
Move re-usable test helpers under `TestUtils`, clean up rehydrated calldata mgmt

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
- Simplifies management of rehydrated calldata.
- Adds and refactors some unit tests.

### CODE REVIEW

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM !

File: hedera-node/src/test/java/com/hedera/services/utils/accessors/SignedTxnAccessorTest.java [hunk 1]
```diff
@@ -173,7 +123,7 @@ void getsCryptoSigMappingFromKnownRationalizedMeta() {
 
     @Test
     void uncheckedPropagatesIaeOnNonsense() {
-        final var nonsenseTxn = buildTransactionFrom(ByteString.copyFromUtf8("NONSENSE"));
+        final var nonsenseTxn = TxnUtils.buildTransactionFrom(ByteString.copyFromUtf8("NONSENSE"));
```
Comment (shemnon) [inline hedera-node/src/test/java/com/hedera/services/utils/accessors/SignedTxnAccessorTest.java:126]: Is name qualifying this static call really needed?  It's generating a large amount of diffs in this file unrelated to the purpose of the change of the PR.  The subsequent reformatting then makes it difficult to see if there are any substantive changes.

**Review by shemnon — DISMISSED** [review]
Two style changes would drop the line count down for this PR in half
- don't migrate imports to wildcard imports, this should trigger the google-java-formatter
- make the re-homed static methods statically imported, rather than adding TestUtils to each instance.

**Review by Nana-EC — APPROVED** [review]
LG

File: hedera-node/src/main/java/com/hedera/services/txns/span/SpanMapManager.java [hunk 2]
```diff
@@ -18,11 +18,7 @@
 import static com.hedera.services.state.merkle.internals.BitPackUtils.codeFromNum;
 import static com.hederahashgraph.api.proto.java.HederaFunctionality.CryptoTransfer;
 import static com.hederahashgraph.api.proto.java.HederaFunctionality.EthereumTransaction;
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_FILE_EMPTY;
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.FILE_DELETED;
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_ETHEREUM_TRANSACTION;
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_FILE_ID;
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.*;
```
[the remark this answers is not in the document]
Comment (tinker-michaelj) [reply hedera-node/src/main/java/com/hedera/services/txns/span/SpanMapManager.java:21]: I'm not sure tbh---`spotlessApply` plugin just gives me this. 🤔

File: hedera-node/src/test/java/com/hedera/services/utils/accessors/SignedTxnAccessorTest.java (hunk 1 above)
Comment (tinker-michaelj) [reply hedera-node/src/test/java/com/hedera/services/utils/accessors/SignedTxnAccessorTest.java]: Used static import.

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**